### ISSUE TITLE
Use One-hot Label

### ISSUE THREAD

**Issue by WingsUpete** [WingsUpete/GNN4Cora#2]
The label tensors are not one-hot encoded. Find time to improve.

**Comment by WingsUpete** [comment]
It seems that in PyTorch, [nn.CrossEntropyLoss](https://pytorch.org/docs/master/generated/torch.nn.CrossEntropyLoss.html) can handle labels that are not one-hot encoded: https://stackoverflow.com/questions/62456558/is-one-hot-encoding-required-for-using-pytorchs-cross-entropy-loss-function, so basically there is no need to one-hot encode it.
